atlasdb-client/src/main/java/com/palantir/atlasdb/table/description/TableMetadata.java

```diff
@@ -27,145 +27,69 @@
+    public CachePriority cachePriority() {
+        return CachePriority.WARM;
     }
 
-    public TableMetadata(NameMetadataDescription rowMetadata,
-                         ColumnMetadataDescription columns,
-                         ConflictHandler conflictHandler,
-                         CachePriority cachePriority,
-                         boolean rangeScanAllowed,
-                         int explicitCompressionBlockSizeKB,
-                         boolean negativeLookups,
-                         SweepStrategy sweepStrategy,
-                         boolean appendHeavyAndReadLight) {
-        this(
-                rowMetadata,
-                columns,
-                conflictHandler,
-                cachePriority,
-                rangeScanAllowed,
-                explicitCompressionBlockSizeKB,
-                negativeLookups,
-                sweepStrategy,
-                appendHeavyAndReadLight,
-                LogSafety.UNSAFE);
+    @Value.Default
+    public boolean rangeScanAllowed() {
+        return false;
     }
 
-    public TableMetadata(NameMetadataDescription rowMetadata,
-                         ColumnMetadataDescription columns,
-                         ConflictHandler conflictHandler,
-                         CachePriority cachePriority,
-                         boolean rangeScanAllowed,
-                         int explicitCompressionBlockSizeKB,
-                         boolean negativeLookups,
-                         SweepStrategy sweepStrategy,
-                         boolean appendHeavyAndReadLight,
-                         LogSafety nameLogSafety) {
-        this.rowMetadata = rowMetadata;
-        this.columns = columns;
-        this.conflictHandler = conflictHandler;
-        this.cachePriority = cachePriority;
-        this.rangeScanAllowed = rangeScanAllowed;
-        this.explicitCompressionBlockSizeKB = explicitCompressionBlockSizeKB;
-        this.negativeLookups = negativeLookups;
-        this.sweepStrategy = sweepStrategy;
-        this.appendHeavyAndReadLight = appendHeavyAndReadLight;
-        this.nameLogSafety = nameLogSafety;
+    @Value.Default
+    public int explicitCompressionBlockSizeKB() {
+        return 0;
     }
 
-    public NameMetadataDescription getRowMetadata() {
-        return rowMetadata;
+    @Value.Default
+    public boolean negativeLookups() {
+        return false;
     }
 
-    public ColumnMetadataDescription getColumns() {
-        return columns;
+    @Value.Default
+    public SweepStrategy sweepStrategy() {
+        return SweepStrategy.CONSERVATIVE;
     }
 
-    public boolean isWriteWriteConflict() {
-        return conflictHandler != ConflictHandler.IGNORE_ALL;
+    @Value.Default
+    public boolean appendHeavyAndReadLight() {
+        return false;
     }
 
-    public ConflictHandler getConflictHandler() {
-        return conflictHandler;
+    @Value.Default
+    public LogSafety nameLogSafety() {
+        return LogSafety.UNSAFE;
     }
```

Comment:
Debatable, and certainly non-blocking: We could probably have gotten away with leaving the method names as they were, and using `@Value.Style(get = {"get*", "is*"})`

Reply:
As much as I dislike the style, minimizing the devbreak makes sense 👍